src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -524,4 +526,16 @@ protected function getErrorMessageForException(Exception $e, array $messages)
             $e->getMessage()
         );
     }
+
+    protected function render($view, array $parameters = [], Response $response = null)
+    {
+        /** @var HookDispatcherInterface $hookDispatcher */
+        $hookDispatcher = $this->get('prestashop.core.hook.dispatcher');
+        /** @var RenderedHook $renderedHook */
```

Comment:
You can use `RenderedHookInterface` instead of the concrete class

Reply:
Also a thing I'm not sure of, but I thought a `RenderedHook` is used for display hooks (as its name would suggest), so it will only contain a string to display whereas here we are aiming for an action hook which would return an array of variables